test/test-ratelim.c

```diff
@@ -256,8 +269,10 @@ check_group_bucket_levels_cb(evutil_socket_t fd, short events, void *arg)
 static void
 group_drain_cb(evutil_socket_t fd, short events, void *arg)
 {
-	bufferevent_rate_limit_group_decrement_read(ratelim_group, cfg_group_drain);
-	bufferevent_rate_limit_group_decrement_write(ratelim_group, cfg_group_drain);
+	if (ratelim_group) {
```

Comment:
It should never be called when ratelim_group is NULL

Reply:
> #1 0x557e063d31e8 in group_drain_cb /home/runner/work/libevent/libevent/test/test-ratelim.c:271:2

I add this check to solve the null pointer dereference here. I don't understand why the code before my modifications didn't cause a crash...something wrong?